Standards/scs-XXXX-vN-mandatory-and-supported-IaaS-services.md

```diff
@@ -46,27 +45,25 @@ Users should always research whether a needed feature is supported in the offere
 
 :::
 
-## Supported OpenStack services
+## Supported IaaS APIs
 
-The following services MAY be present in SCS-compliant IaaS deployment and are considered in the SCS standards.
+The following IaaS APIs MAY be present in SCS-compliant IaaS deployment, e.g. implemented thorugh the corresponding OpenStack services, and are considered in the SCS standards.
```

Comment:
No these APIs are truely optional, but in contrast to the unsupported APIs, we know they are likely to be used and such consider them in standards and (except for Masakari, if I understood @berendt correctly) even tested for integration.

I can edit the sentece to better reflect the wide but optional usage of these APIs, which is the reason for us to also think of them in standards and maybe sometimes test their integration?

Reply:
Is it because of the last part of the sentence? "... are considered in the SCS standards"? This is, because we need (for some standards) to consider MANY possible cloud configurations. The role standard for example with the new RBAC roles must decide between old and new roles, which has to be rolled out throughout all services in a deployment. If at some point we want to use the new RBAC system, all servies need to be able to do this, and while the mandatory service already are in a good state, some supported services are not. SO we do have to consider all only supported services in this standard, because many clouds use them, and we don't want to break them.
The mandatory Services are more of a minimal list of services, that are needed to have an scs-compliant cloud.

I hope this explains it a little bit better.

Reply:
No, sorry, I think I didn't make clear what I meant:

In line 50 we have:

```suggestion
The following IaaS APIs MAY be present [..]
```

but in line 52, immediately following this we have:

```
| **Mandatory** API | corresponding OpenStack Service | description |
```

which imho is a contradiction in the heading.

Reply:
Uh I made a mistake when adding the first column to the table. It should be "Supported API". Thank you for pointing it out, I changed it.